tests/api_connexion/endpoints/test_dag_endpoint.py

```diff
@@ -418,7 +418,7 @@ def test_should_response_200_for_null_start_date(self, url_safe_serializer):
             "tags": [],
             "template_searchpath": None,
             "timetable_description": None,
-            "timezone": "Timezone('UTC')",
+            "timezone": "UTC",
```

Comment:
Well, change in default type for UTC change API response and instead of __repr__ it starts return Timezone